### PR TITLE
[CVSRP-6150] remove unused feature switches

### PR DESCRIPTION
**Pull request by peckoverHMRC**

### CODE REVIEW

**Comment by peckoverHMRC** [comment]
https://github.com/hmrc/app-config-qa/pull/13598
https://github.com/hmrc/app-config-staging/pull/9558
https://github.com/hmrc/app-config-production/pull/13251

**Comment by peckoverHMRC** [comment]
test this please

File: app/navigation/Navigator.scala [hunk 1]
```diff
@@ -468,7 +462,7 @@ class Navigator @Inject()(implicit frontendAppConfig: FrontendAppConfig)
   private def annualPayAmountRoutes: UserAnswers => Call =
     userAnswers =>
       if (isPhaseTwoOnwards(userAnswers)) {
```
Comment (mywyau) [inline app/navigation/Navigator.scala:464]: I know it's fine as is already just wondering if it's worth re writing the nested if else's into a match case 
```
  private def annualPayAmountRoutes: UserAnswers => Call = { userAnswers =>
    {
      (isPhaseTwoOnwards(userAnswers), isMayExtensionOnwards(userAnswers)) match {
        case (true, true)  => routes.HasEmployeeBeenOnStatutoryLeaveController.onPageLoad()
        case (true, false) => routes.PartTimeQuestionController.onPageLoad()
        case _             => phaseOneAnnualPayAmountRoute(userAnswers)
      }
    }
  }
```

Comment (peckoverHMRC) [reply app/navigation/Navigator.scala:464]: Good shout that's easier to read

**Comment by mywyau** [comment]
https://github.com/hmrc/coronavirus-jrs-calculator-acceptance-tests/pull/129 <- ATs

**Review by mywyau — APPROVED** [review]

**Pull request merged**